setup.py

```diff
@@ -144,11 +120,12 @@
     "packaging>=20.0",
     "parameterized",
     "phonemizer",
+    "Pillow>=10.0.1,<=15.0",
     "protobuf",
     "psutil",
     "pyyaml>=5.1",
     "pydantic",
-    "pytest>=7.2.0,<8.0.0",
```

Comment:
The pin on `pytest` was added to fix CI (https://github.com/huggingface/transformers/pull/28758) -- removing to test whether it is still needed